scripts/make_wheel.sh

```diff
@@ -291,69 +291,99 @@ package_wheel() {
     cd ..
   fi
 
-  cd ${WORKSPACE}/${build_type}/src/python
-  dist_type="bdist_wheel"
+  # helper function defined within function package_wheel
+  function package_wheel_helper {
+    local minimal=$1
+    local version_modifier=$2
 
-  VERSION_NUMBER=`${PYTHON_EXECUTABLE} -c "from turicreate.version_info import version; print(version)"`
-  ${PYTHON_EXECUTABLE} setup.py ${dist_type} # This produced an wheel starting with turicreate-${VERSION_NUMBER} under dist/
+    cd ${WORKSPACE}/${build_type}/src/python
 
-  cd ${WORKSPACE}
+    if [[ "$minimal" -eq 1 ]] ; then
+      if [[ "$(uname -s)" == "Darwin" ]] ; then
+        sed -i "" 's/^USE_MINIAL = False$/USE_MINIAL = True/g' turicreate/_deps/minimal_import.py
```

Comment:
Should this be USE_MINIMAL ?

Reply:
Yeah, that's my typo. I use `USE_MINIAL` everywhere. Good Catch!